packages/mermaid/src/docs/config/math.md

```diff
@@ -0,0 +1,62 @@
+# Math Configuration
```

Comment:
We should add a version number here. 
Something like. 

```suggestion
# Math Configuration (since v10.3)
```

Reply:
Or 
`( v10.3+)

Reply:
I got the idea for https://github.com/mermaid-js/mermaid/pull/4612 from here. 
So, you can add `(v<MERMAID_RELEASE_VERSION>+)` now.